src/auth/ejabberd_auth_rdbms.erl

```diff
@@ -285,6 +286,11 @@ remove_user(HostType, LUser, LServer) ->
     end,
     ok.
 
+-spec remove_domain(mongooseim:host_type(), jid:lserver()) -> ok.
+remove_domain(HostType, LServer) ->
+    execute_successfully(HostType, auth_remove_domain, [LServer]),
```

Comment:
Should we use here LServer or Domain? In `mod_roster` I changed it to Domain.

Reply:
Not sure, all the functions in this module use `LServer`. I simply wanted to remain consistent with the rest of this module.